src/app/clusters/application-launcher-server/application-launcher-server.cpp

```diff
@@ -222,10 +387,72 @@ bool emberAfApplicationLauncherClusterHideAppRequestCallback(app::CommandHandler
     EndpointId endpoint = commandPath.mEndpointId;
     auto & application  = commandData.application;
 
+    ApplicationLauncherApplication appInput;
+    appInput.catalogVendorId = application.catalogVendorId;
+    appInput.applicationId   = application.applicationId;
+
     Delegate * delegate = GetDelegate(endpoint);
     VerifyOrExit(isDelegateNull(delegate, endpoint) != true, err = CHIP_ERROR_INCORRECT_STATE);
 
     {
+#if CHIP_DEVICE_CONFIG_APP_PLATFORM_ENABLED
+        // if the feature flag is APP_Plaform, then this is a call to endpoint 1 to stop an app
+        //  1. Find target Content App
+        //  3. If this was the current app then hide it
+        //  2. Set Content App status (basic cluster) to ACTIVE_VISIBLE_NOT_FOCUS
+        //  4. Call stop app command on Content App
+        if (delegate->HasFeature(chip::app::Clusters::ApplicationLauncher::ApplicationLauncherFeature::kApplicationPlatform))
+        {
+            ChipLogError(Zcl, "ApplicationLauncher has content platform feature");
+            ContentApp * app = chip::AppPlatform::AppPlatform::GetInstance().GetContentAppByAppId(appInput);
+            if (app == NULL)
```

Comment:
```suggestion
            if (app == nullptr)
```
and so on with NULL and namespaces, just like in the other functions.

Reply:
fixed